core/src/main/java/io/aiven/klaw/helpers/HandleDbRequests.java

```diff
@@ -363,4 +363,32 @@ List<Map<String, String>> selectAclsCountByTeams(
   int getAllTopicsCountInAllTenants();
 
   int findAllComponentsCountForUser(String userName, int tenantId);
+
+  List<Topic> getAllTopics();
+
+  List<TopicRequest> getAllTopicRequests();
+
+  List<KafkaConnectorRequest> getAllConnectorRequests();
+
+  List<KwKafkaConnector> getAllConnectors();
+
+  List<Acl> getAllSubscriptions();
+
+  List<AclRequests> getAllAclRequests();
+
+  List<SchemaRequest> getAllSchemaRequests();
+
+  List<MessageSchema> selectAllSchemas();
+
+  List<Team> selectTeams();
+
+  List<RegisterUserInfo> getAllRegisterUsersInfo();
+
+  List<Env> selectEnvs();
+
+  List<ActivityLog> getAllActivityLog();
+
+  List<KwProperties> selectKwProperties();
+
+  List<KwClusters> getClusters();
```

Comment:
Only note here is should we decide on a common way of naming these get vrs select ?

Reply:
Agree, some are starting with get and some with select. Created an issue for this refactoring on the whole interface.
https://github.com/aiven/klaw/issues/959